functests/utils/images/images.go

```diff
@@ -13,7 +13,7 @@ func init() {
 	cnfTestsImage = os.Getenv("CNF_TESTS_IMAGE")
 
 	if cnfTestsImage == "" {
-		cnfTestsImage = "cnf-tests:4.5"
+		cnfTestsImage = "cnf-tests:4.7"
```

Comment:
Not really, see https://github.com/openshift-kni/performance-addon-operators/pull/433/files#diff-a11540b25a190b62a4789801be3fa3b70b91847dc903680998aa8a5837550646R215, before it used the oslat from our custom image instead of cnf-tests

Reply:
Aha, I missed that, now it does makes sense!